### PR TITLE
Documentation without setup

### PR DESCRIPTION
**Pull request by musyokamuasya**

### CODE REVIEW

**Comment by justtabasumu** [comment]
@musyokamuasya I feel that this is just a duplication of the documentation from jambo, please make sure the readme is custom to this library/repositody

**Comment by mawindavic** [comment]
@musyokamuasya Also add how to use the lib and attach screen how it looks

File: readme.md [hunk 1]
```diff
@@ -2,4 +2,128 @@
 [![](https://jitpack.io/v/MamboBryan/calendar-row.svg)](https://jitpack.io/#MamboBryan/calendar-row)
 
 
-This is a work in progress and it will soon be documented
\ No newline at end of file
+<div align="center">
+TODO -> Get the logo for the lib
```
Comment (MamboBryan) [inline readme.md:6]: I don't think it has one, it's just a standard library

File: readme.md [hunk 2]
```diff
@@ -2,4 +2,128 @@
...
+    }
+}
+```
+
+* Add Jambo dependency
+
+In your app module find `build.gradle` and add :
+
+```groovy
+  implementation 'com.github.tabasumu:calendar-row:$version'
```
Comment (MamboBryan) [inline readme.md:93]: Have you tried this in a separate app and it worked?

**Review by MamboBryan — CHANGES_REQUESTED** [review]
I have added some few comments

**Pull request closed without merging**